.github/workflows/dockerimage.yml

```diff
@@ -71,12 +71,12 @@ jobs:
         export HUMHUB_VERSION=$(awk -e '$0 ~ /^([0-9\.]+) ${{ matrix.version }}/ {print $1}' versions.txt)
         echo "HUMHUB_VERSION=$HUMHUB_VERSION" >> $GITHUB_ENV
     - name: Login to registry
-      run: echo "${{ secrets.PAT }}" | docker login ghcr.io --username mriedmann --password-stdin
+      run: echo "${{ secrets.PAT }}" | docker login ghcr.io --username ${GITHUB_REPOSITORY_OWNER} --password-stdin
     - name: Pull images 
       run: |
-        docker pull ghcr.io/mriedmann/humhub:${GITHUB_REF_NAME_SLUG}-${HUMHUB_VERSION}-allinone
+        docker pull ghcr.io/${GITHUB_REPOSITORY_OWNER}/humhub:${GITHUB_REF_NAME_SLUG}-${HUMHUB_VERSION}-allinone
     - name: replace image
-      run: sed -i "s/docker.io\/mriedmann\/humhub:latest/ghcr.io\/mriedmann\/humhub:${GITHUB_REF_NAME_SLUG}-${HUMHUB_VERSION}-allinone/g" docker-compose.yml
```

Comment:
please be careful here: As long as the docker-compose file is not changed the first part should stay static. From my point of view, this would break with the current compose file.

Reply:
sorry, wrong replace all.